config/dev.exs

```diff
@@ -55,5 +55,17 @@ config :elixir_jobs, ElixirJobsWeb.Guardian,
   issuer: "ElixirJobs",
   secret_key: "MY_D3V_K3Y"
 
+config :elixir_jobs, ElixirJobsWeb.Mailer,
```

Comment:
In development, I'd rather use the `Bamboo.LocalAdapter` adapter, which allows to integrate this: [EmailPreviewPlug](https://hexdocs.pm/bamboo/Bamboo.EmailPreviewPlug.html).

It makes the development much easier.